pkg/controller/secrets_test.go

```diff
@@ -0,0 +1,193 @@
+// Copyright (C) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package controller
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/verrazzano/verrazzano-operator/pkg/constants"
+	. "github.com/verrazzano/verrazzano-operator/pkg/managed"
```

Comment:
Are you using import dot notation for this reason?  https://github.com/golang/go/wiki/CodeReviewComments#import-dot 
If not, we should use the package name since it is not obvious that the methods are test methods, like GetManagedClusterConnections().  It will be even better when it is testutil. GetManagedClusterConnections()

Reply:
Will fix.

Reply:
Fixed